releasenotes/notes/fix-9127-a16104e9efd4b8f2.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Fixes issue 9127
```

Comment:
I would include a descriptive message. Something like
In `AzureOpenAIGenerator` and `AzureOpenAIChatGenerator`, ensure that the `max_retries` initialization parameter is taken into account when not null.